src/main/webapp/WEB-INF/tags/ksr/securityRequestRoleQualifications.tag

```diff
@@ -8,8 +8,10 @@
 <c:set var="securityRequestRole" value="${KualiForm.document.securityRequestRoles[securityRequestRoleIndex]}" />
 
 <c:set var="securityRequestRoleQualificationHdr" value="${securityRequestRole.newRequestRoleQualification}" />
+<c:set var="roleQualifierReadOnly" value="${!securityRequestRole.allowKSRToManageQualifications}"/>

```

Comment:
should roleQualifierReadOnly be renamed to roleQualificationReadOnly?

Reply:
ok